examples/webots/vacuum/vacuum_simple.scenic

```diff
@@ -0,0 +1,146 @@
+    positionStdDev: (0.05, 0.5 ,0)
+    orientationStdDev: (5 deg, 0, 0)
+    color: [51, 51, 255]
+
+class CoffeeTable(WebotsObject):
+    webotsAdhoc: {'physics': False}
+    shape: MeshShape.fromFile(Path(localPath(".")).parent.parent.parent / "tools" / "meshes" / "coffee_table.obj.bz2")
+    width: 1.5
+    length: 0.5
+    height: 0.4
+    positionStdDev: (0.05, 0.05 ,0)
+    orientationStdDev: (5 deg, 0, 0)
+    color: [103, 71, 54]
+
+class Toy(WebotsObject):
+    webotsAdhoc: {'physics': True}
+    shape: Uniform(BoxShape(), CylinderShape(), ConeShape(), SpheroidShape())
+    width: 0.1
+    length: 0.1
+    height: 0.1
+    density: 100
+    color: [255, 128, 0]
+
+class BlockToy(Toy):
+    shape: BoxShape()
+
+## Scene Layout ##
+
+# Create room region and set it as the workspace
+room_region = RectangularRegion(0 @ 0, 0, 5.09, 5.09)
+workspace = Workspace(room_region)
+
+# Create floor and walls
+floor = new Floor
+wall_offset = floor.width/2 + 0.04/2 + 1e-4
+right_wall = new Wall at (wall_offset, 0, 0.25), facing toward floor
+left_wall = new Wall at (-wall_offset, 0, 0.25), facing toward floor
+front_wall = new Wall at (0, wall_offset, 0.25), facing toward floor
+back_wall = new Wall at (0, -wall_offset, 0.25), facing toward floor
+
+# Place vacuum on floor
+ego = new Vacuum on floor,
+    with customData str(random.getrandbits(32))
+
+# Create a "safe zone" around the vacuum so that it does not start stuck
+safe_zone = CircularRegion(ego.position, radius=1)
+
+# Create a dining room region where we will place dining room furniture
+dining_room_region = RectangularRegion(1.25 @ 0, 0, 2.5, 5).difference(safe_zone)
+
+# Place a table with 4 chairs around it, one which is knocked over
+dining_table = new DiningTable contained in dining_room_region, on floor,
+    facing Range(0, 360 deg)
+
+chair_1 = new DiningChair behind dining_table by -0.1, on floor,
+                facing toward dining_table, with regionContainedIn dining_room_region
+chair_2 = new DiningChair ahead of dining_table by -0.1, on floor,
+                facing toward dining_table, with regionContainedIn dining_room_region
+chair_3 = new DiningChair left of dining_table by -0.1, on floor,
+                facing toward dining_table, with regionContainedIn dining_room_region
+
+# Create a living room region where we will place living room furniture
+living_room_region = RectangularRegion(-1.25 @ 0, 0, 2.5, 5).difference(safe_zone)
+
+couch = new Couch ahead of left_wall by 0.335,
+            on floor, facing away from left_wall
+
+coffee_table = new CoffeeTable ahead of couch by 0.336,
+            on floor, facing away from couch
+
+# Add some noise to the positions of the couch and coffee table
+mutate couch
+mutate coffee_table
+
+# Spawn some toys
+for _ in range(globalParameters.numToys):
+    new Toy on floor.topSurface
+
+## Simulation Setup ##
+terminate after globalParameters.duration * 60 seconds
+record (ego.position.x, ego.position.y) as VacuumPosition
```

Comment:
```suggestion
record (ego.x, ego.y) as VacuumPosition
```
(assuming you really need a 2-tuple here; otherwise I'd just do `ego.position`)

Reply:
We're concerned with 2D coverage and we pass the values to shapely, so I think it's clearest to drop the Z component.